Source/core/inspector/InspectorInstrumentation.h

```diff
@@ -164,16 +164,25 @@ inline InstrumentingAgents* instrumentingAgentsFor(CSSStyleDeclaration* declarat
     return declaration ? instrumentingAgentsFor(declaration->parentStyleSheet()) : 0;
 }
 
+#ifdef DISABLE_INSPECTOR
+inline void addMessageToConsoleImpl(InstrumentingAgents*, ConsoleMessage*){}
```

Comment:
Since it's a header and included by multi files, without "inline" there will be a "redefinition of function" error.